### DISCUSSION TITLE
Create linked view with API

### DISCUSSION THREAD

**Discussion by PierreVannier** [ramnes/notion-sdk-py#201] [Q&A]
Hi folks, 

I wonder if you can create a linked view / database with the API with certain criteria, then prepend this block at the top of a given page ?

**Comment by ramnes** [comment]
Coucou Pierre ! :)

AFAIK the Notion API doesn't expose links.

From https://developers.notion.com/docs/working-with-databases:

> The Notion API does not work with linked databases. When sharing a database with your integration, make sure it's the source one!

**Reply by PierreVannier** [reply]
Salut Guillaume,
Damned… I wanted to update a lot of pages and add a filtered linked view related to the page…
This is really sad because there is no way to add a linked view with a filter that reference a field in the given page…
Let’s say you got a companies database, I want to create a list of actions related to each company on its own page. Can’t do that 😔

**Reply by ramnes** [reply]
Can you be a little more specific as to what you try to achieve? Maybe there's another way.

**Reply by PierreVannier** [reply]
Yes, sorry.
I've got a database called **companies**.
Ive got another related database called **clients** (actual people).
One company can have 1/n clients.

I've a more generic database which is called "**actions**" which stores all actions done with clients. (Actions is related to clients).

On each company page I would have liked to display as a block an linked database (filtered) which would have listed all the actions related to clients who are related to the company were on.
As I've got more than 400 hundreds companies I would like to do it the more automatically that I could.
